apps/Ruffle/install

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+version=nightly-2023-03-02 # dummy variable to make Pi-Apps update the app every new version
```

Comment:
This is not automatic, it needs a github actions script made, such as the ones you can find in .github/workflows/updates folder.
Documentation here: https://pi-apps.io/wiki/development/DOCUMENTATION/#automatic-app-updaters

Reply:
Let me update that documentation. While "webVer" is still supported, it isn't the suggested variable name anymore to make it easier on developers.